### PR TITLE
Descriptor cluster must be handled by the AttributeAccessInterface in the bridge-app (#11182)

### LINKED ISSUE #11182: Descriptor cluster must be handled by the AttributeAccessInterface in the bridge-app
#### Problem
The handling of the descriptor cluster in the bridge-app should be done completely via the AttributeAccessInterface. 
The implementation in the bridge-app can be removed.

 #### Proposed Solution
Implement as described.

### PR DESCRIPTION
**Pull request by pjzander-signify**
#### Problem
The handling of the descriptor cluster in the bridge-app should be done completely via the AttributeAccessInterface.
The implementation in the bridge-app can be removed.

#### Change overview
- The cluster revision attribute is also returned by descriptor cluster implementation. Now all the attributes are implemented in the descriptor cluster.
- The handling of reading the descriptor cluster attributes is removed completely from the bridge-app since it is not necessary anymore.

#### Testing
- Manual tested with the chip-tool and the bridge-app.
- Passed all Github workflows.

### CODE REVIEW

**Review by andy31415 — APPROVED** [review]

File: examples/bridge-app/linux/main.cpp [hunk 1]
```diff
@@ -325,25 +319,6 @@ EmberAfStatus HandleWriteOnOffAttribute(Device * dev, chip::AttributeId attribut
     return EMBER_ZCL_STATUS_SUCCESS;
 }
```
Comment (mrjerryjohns) [inline examples/bridge-app/linux/main.cpp:320]: Can't comment on the text above, but _I think_ you should be able to delete `DECLARE_DYNAMIC_ATTRIBUTE_LIST_BEGIN(descriptorAttrs)` as well as the instantiation of `descriptorAttrs` on your dynamic endpoint.

**Review by mrjerryjohns — APPROVED** [review]

File: examples/bridge-app/linux/main.cpp (hunk 1 above)
Comment (pjzander-signify) [reply examples/bridge-app/linux/main.cpp:320]: If I delete the DECLARE_DYNAMIC_ATTRIBUTE_LIST_BEGIN(descriptorAttrs), the dynamic endpoint will not contain the descriptor cluster.

File: examples/all-clusters-app/all-clusters-common/all-clusters-app.zap [hunk 2]
```diff
@@ -859,7 +859,7 @@
               "mfgCode": null,
               "side": "server",
               "included": 1,
-              "storageOption": "RAM",
+              "storageOption": "External",
```
Comment (bzbarsky-apple) [inline examples/all-clusters-app/all-clusters-common/all-clusters-app.zap:862]: Should make this change to all .zap files that include Descriptor, no?  And to all endpoints that have Descriptor on them.

**Review by bzbarsky-apple — APPROVED** [review]

File: examples/all-clusters-app/all-clusters-common/all-clusters-app.zap (hunk 2 above)
Comment (pjzander-signify) [reply examples/all-clusters-app/all-clusters-common/all-clusters-app.zap:862]: I updated all .zap file.

**Comment by bzbarsky-apple** [comment]
@msandstedt @saurabhst @jepenven-silabs @jmartinez-silabs @LuDuda @Damian-Nordic Please take a look?

**Review by jmartinez-silabs — APPROVED** [review]

**Pull request merged**